src/app/core/services/search-criteria/search-criteria.service.ts

```diff
@@ -10,6 +10,12 @@ import { Immutable } from 'src/app/core/models/immutable';
 import { GeoHazard } from 'src/app/modules/common-core/models';
 import { UrlParams } from './url-params';
 
+enum ORDER_BY_TYPE {
+'changeTime', 'obsTime' , 'DtObsTime' , 'DtChangeTime'
```

Comment:
Jeg lurer på om du ville være mer tjent med å splitte denne i to, en for lovlige url-parametre og en for lovlige sorteringsvalg i søkekriteria. Da unngår du at utvikler roter med hvilken hun skal bruke. Jeg tror også koden din ville bli enklere om du brukte typer i stedet for enum, f.eks.:
`type SearchCriteriaOrderBy = 'DtObsTime' | 'DtChangeTime';`